src/libcollections/btree/node.rs

```diff
@@ -1357,9 +1237,11 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Internal>, marker::
                     right.correct_childrens_parent_links(count, count + right_len + 1);
 
                     move_edges(left, new_left_len + 1, right, 0, count);
-                },
-                (ForceResult::Leaf(_), ForceResult::Leaf(_)) => { }
-                _ => { unreachable!(); }
+                }
+                (ForceResult::Leaf(_), ForceResult::Leaf(_)) => {}
+                _ => {
+                    unreachable!();
+                }
```

Comment:
Rustfmt won't do this for you since in some cases it can make the code stop compiling. However, if you change this to not have a semi-colon, Rustfmt will leave it on one line.